wp-content/themes/pub/wporg-learn-2024/templates/taxonomy-learning-pathway.html

```diff
@@ -1,45 +1,11 @@
-<!-- wp:template-part {"slug":"header","className":"has-display-contents"} /-->
+<!-- wp:template-part {"slug":"header","className":"has-display-contents","tagName":"div"} /-->
```

Comment:
This avoids invalid html where the global `header` and `footer` elements are nested in `header` and `footer` elements created by these template parts